eth/api.go

```diff
@@ -439,6 +440,19 @@ func (s *PrivateAccountAPI) NewAccount(password string) (common.Address, error)
 	return common.Address{}, err
 }
 
+func (s *PrivateAccountAPI) ImportAccount(privkey string, password string) (common.Address, error) {
+	hexkey, err := hex.DecodeString(privkey)
+	if err != nil {
+		return common.Address{}, err
+	}
+
+	acc, err := s.am.ImportECDSA(crypto.ToECDSA(hexkey), password)
+	if err == nil {
+		return acc.Address, nil
+	}
+	return common.Address{}, err
+}
```

Comment:
Maybe we should name this method differently. ImportAccount makes it sound like it takes an encrypted JSON key.
ImportRawKey might be a better name.